fvtest/compilertriltest/VectorTest.cpp

```diff
@@ -39,7 +39,11 @@ TEST_F(VectorTest, VDoubleAdd) {
     auto trees = parseString(inputTrees);
 
     ASSERT_NOTNULL(trees);
-
+    // This test is currently disabled on Z platforms because not all Z platforms
+    // have vector support. Determining whether a specific platform has the support
+    // at runtime is currently not possible in tril. So the test is being disabled altogether
+    // on Z for now.
+#ifndef TR_TARGET_S390
```

Comment:
Could we please add a TODO and in it reference the Issue which will eventually fix this problem? Otherwise this comment will get lost in the sands of time (and code).

Reply:
Okay, so looks like #1843 is the actual issue that should resolve this problem once and for all. So I'll add the "Closes: #1941" tag for this problem as well since that problem should technically be solved by this PR.